packages/hooks/src/createDidHook.ts

```diff
@@ -1,28 +1,26 @@
 /**
  * Internal dependencies
  */
-import validateHookName from './validateHookName.js';
+import validateHookName from './validateHookName';
+import type { Hooks, StoreKey } from '.';
 
 /**
- * @callback DidHook
  *
  * Returns the number of times an action has been fired.
  *
- * @param {string} hookName The hook name to check.
- *
- * @return {number | undefined} The number of times the hook has run.
  */
+export type DidHook = ( hookName: string ) => number | undefined;
```

Comment:
Same here about retaining the argument docs

```suggestion
export type DidHook = (
	/**
	 * The hook name to check.
	 */
	hookName: string
) => number | undefined;
```